Dockerfile

```diff
@@ -1,3 +1,5 @@
-FROM jupyterhub/jupyterhub:1.0
+FROM jupyterhub/jupyterhub:1.0.0
```

Comment:
Thanks for pinning that.  1.1.0 is available if you feeling lucky :D.  I am fine with 1.0.0.